README.md

```diff
@@ -1,22 +1,35 @@
-# Unity Multiplayer Docs
+# Unity Multiplayer Networking Docs
 
-Welcome to the Unity Multiplayer Technology (MTT) Documentation. This site includes the MLAPI, Unity Transport Package, and other tools and features supporting MTT.
+Welcome to the Unity Multiplayer Networking Documentation. This site includes the MLAPI, Unity Transport Package, Boss Room Sample Coop, and other tools and features.
 
-The documentation site is built using [Docusaurus 2](https://v2.docusaurus.io/). 
+The documentation site is built using [Docusaurus 2](https://v2.docusaurus.io/) using [Yarn](https://yarnpkg.com/en/) and [Node.js](https://nodejs.org/en/download/). 
 
+[![Website](https://img.shields.io/website?url=https%3A%2F%2Fdocs-multiplayer.unity3d.com%2F)](https://docs-multiplayer.unity3d.com/)
 [![Forums](https://img.shields.io/badge/unity--forums-multiplayer-blue)](https://forum.unity.com/forums/multiplayer.26/)
 [![Discord](https://img.shields.io/discord/449263083769036810.svg?label=discord&logo=discord&color=informational)](https://discord.gg/FM8SE9E)
 
 ## Contributing Content
 
-See the [Contribution Guide](https://github.com/Unity-Technologies/com.unity.multiplayer.docs/wiki) for details. A [template file](https://github.com/Unity-Technologies/com.unity.multiplayer.docs/blob/master/docs/template.md) (/docs/template.md) is included in the repo to understand markdown formatting and options.
+Always cut branches from **master** branch. 
+
+See the [Contribution Guide](https://github.com/Unity-Technologies/com.unity.multiplayer.docs/wiki) to learn about writing and submitting content, our GitHub labels, and more. A [template file](https://github.com/Unity-Technologies/com.unity.multiplayer.docs/blob/master/docs/template.md) (/docs/template.md) is included in the repo to understand markdown formatting and options.
+
+## Tools Used
+
+Our site uses Docusaurus with Yarn commands. Always use Yarn commands, to update, build, and serve the server locally. If you need a preview without building locally, the Unity Doc Team can get you a preview!****
 
 ## Installation
 
 The first time you clone the repo, run the following command to install required node modules into the local cloned directory. You only need to run this once.
 
 ```console
-npm install
+yarn install
```

Comment:
so this works again? 🙂

Reply:
also `yarn` stuff is expected to work fine now, I think we should get rid of `yarn.lock` entry in the `.gitignore` file. what do you think?

Reply:
When this is merged, all commands are yarn! 🚀